tests/flow/test_distinct.py

```diff
@@ -0,0 +1,103 @@
+from redis_base import RedisGraphTestBase
+
+class ReturnDistinctFlowTest_1(RedisGraphTestBase):
+    @classmethod
+    def createCommands(cls):
```

Comment:
why createCommand in ReturnDistinctFlowTest_1 differs from createCommand in ReturnDistinctFlowTest_2 ?

Reply:
The only difference is the order of the resultset. These tests take note of that difference. However, if the order will change in the future and the test fails, what should we do? My intuition is to keep the 2nd test, even though the change in order is not directly related to the DISTINCT operation.